src/engraving/layout/v0/tlayout.cpp

```diff
@@ -3652,6 +3650,19 @@ void TLayout::layout(Ornament* item, LayoutContext& ctx)
     cueNoteChord->setPosX(minDist);
 }
 
+void TLayout::layoutTrillCueNote(Ornament* item, LayoutContext& ctx)
+{
+    Chord* cueNoteChord = item->cueNoteChord();
+
+    if (!cueNoteChord) {
+        return;
+    }
+
+    Note* cueNote = cueNoteChord->notes().front();
```

Comment:
Never write code like this. Always check for emptiness

Reply:
Sorry, with chords I tend to forget because chords are not allowed to have zero notes (there are several other checks and asserts around the code to prevent that). You're right though, better be safe